finalPrep/annotations.js

```diff
@@ -2,41 +2,60 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+// Groups the properties of the Build a Bear.
```

Comment:
These are actually the `parameters` of the `buildABear` function.  They'll become property values later on.